.github/ISSUE_TEMPLATE/NextJS Bug.md

```diff
@@ -0,0 +1,79 @@
+---
+name: Bug report
+about: Create a bug report for Next js 
+title: ""
+labels: 'bug'
+assignees: ''
+
+---
```

Comment:
Remove the table. All of this is already on github we don't need a separate table for it.

Reply:
As it's a Nextjs issue, can you also add the label "Next" to the labels? I am pretty sure we can add multiple labels...